crates/builder/tests/tx_poller_test.rs

```diff
@@ -0,0 +1,78 @@
+mod tests {
+    use std::str::FromStr;
+
+    use alloy_consensus::{SignableTransaction, TxEip1559, TxEnvelope};
+    use alloy_primitives::{bytes, Address, TxKind, U256};
+    use alloy_signer::SignerSync;
+    use alloy_signer_local::PrivateKeySigner;
+
+    use builder::config::BuilderConfig;
+    use builder::tasks::{block::BlockBuilder, tx_poller};
+
+    #[ignore = "integration test"]
```

Comment:
I didn't expand on it in previous comment, but this was done because the Github Action uses the Rust base action which runs all tests, and these integration tests are not setup to run in CI yet. To run ignored tests locally instead, run `cargo test -- --ignored`. 

I admit this isn't ideal but the other option requires changing to a custom Github Action instead of using Rust base. This is probably necessary in the long run, but in the short term I have taken the same approach in `signet-node` for the same reason. It might be worth looking into standardizing on our own Rust base action, though 🤔